portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +26,36 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<String> messages;
```

Comment:
Add a comment as to what this variable is.

Reply:
Yup, done! Can you check it to see if it's descriptive enough?